src/python/turicreate/toolkits/image_analysis/__init__.py

```diff
@@ -7,6 +7,6 @@
 from __future__ import division as _
 from __future__ import absolute_import as _
 
-__all__ = ["image_analysis"]
+# __all__ = ["image_analysis"]
 
-from . import image_analysis
+from .image_analysis import *
```

Comment:
We generally try to void `import *`. I think it's generally considered bad practice.

Reply:
Then only reason I did this was because:
1. If I don't do this I don't think I can access the functions from inside image_similarity.py or image_classifier.py.
2. Also, I looked up `__init__.py` file in both image_similarity and image_classifier and there I saw `from .image_similarity import *` and `from .image_classifier import *` respectively and that's why I did this.